iml-manager-cli/src/server.rs

```diff
@@ -31,6 +31,12 @@ pub struct AddHosts {
     /// The profile to deploy to each host
     #[structopt(short = "p", long = "profile")]
     profile: String,
+    /// Prompt to continue if command fails
+    #[structopt(long)]
+    prompt: bool,
+    /// Always continue if command fails
+    #[structopt(long)]
+    force: bool,    
```

Comment:
It looks line the new options (`--force/--prompt`) are not mutually exclusive.

Reply:
You are right. Structopt doesn't have mutex options, at least in official documentation. I could add a manual check, but it splits parameter checking and reporting in two places.

Reply:
I think manual check should be added, specifying `--force` and `--prompt` makes no sense. It could happen by accident, need to protect against that.

Reply:
In general, the last seen option should override any previous.

Reply:
I'd make an error first, if some conflicting options provided. The lesser implicit behavior - the earlier errors can be detected, hence the easier to maintain the things.

Reply:
I think the ideal could be something like `--on-failure=(ask|force|stop)` :)

Reply:
Good point; since the options are mutually exclusive, let's go with an enum approach which makes multiple selections impossible. 

Like @ip1981 said, if we go with something like:

`--on-failure=(prompt|force|)` we will catch either case. I don't think we need stop as not specifying the flag at all can mean stop.

Reply:
Good idea, I'll try to implement om weekend.

Reply:
I was thinking about "stop" (which would be the default) for completeness, so that there could be a way to override anything (even back to the default). 
Maybe implementation can be simpler too.

Reply:
I will have to implement Stop, at least in the enum, because I need a state for current behaviour. Implementation as cli parameter will depend on structopt capabilities.

Reply:
Well, I have found very simple solution. Structopt can directly use clap methods, and clap surely has mutex options. So I just use clap from structopt directive.

Reply:
The result is "error: The argument '--force' cannot be used with '--prompt'" which is exactly what we want.